ding/entry/tests/test_serial_entry_algo.py

```diff
@@ -2,8 +2,11 @@
 import time
 import os
 import torch
+import subprocess
 from copy import deepcopy
+from kubernetes import config, client, watch, dynamic
```

Comment:
move import `kubernetes` to the specific test function

Reply:
done